nupic/research/frameworks/vernon/handlers.py

```diff
@@ -1059,14 +1059,352 @@ def get_execution_order(cls):
+        - slow parameters: Updated during the outer loop (meta-training testing)
+    """
+
+    def setup_experiment(self, config):
+        """
+        Configure the experiment for training
+
+        :param config: Dictionary containing the configuration parameters, most of
+        which are defined in SupervisedExperiment, but some of which are specific to
+        MetaContinualLearningExperiment
+
+            - experiment_class: Class used to run experiments, specify
+            `MetaContinualLearningExperiment` for meta-continual learning
+            - adaptation_lr: Learning rate used to update the fast parameters during
+            the inner loop of the meta-training phase
+            - tasks_per_epoch: Number of different classes used for training during the
+            execution of the inner loop
+            - slow_batch_size: Number of examples in a single batch used to update the
+            slow parameters during the meta-training testing phase
+            - replay_batch_size: Number of examples in a single batch sampled from all
+            data, also used to train the slow parameters
+        """
+
+        # Simplify loading
+        config["model_args"]["num_classes"] = config.get("num_classes", )
+
+        super().setup_experiment(config)
+        self.train_slow_loader = self.train_loader
+        self.train_fast_loader, self.val_fast_loader, self.train_replay_loader = \
+            self.create_fast_slow_loaders(config)
+
+        self.epochs_to_validate = []
+        self.tasks_per_epoch = config.get("tasks_per_epoch", 1)
+        self.num_classes = min(
+            config.get("num_classes", 50),
+            self.train_fast_loader.sampler.num_classes
+        )
+
+        self.adaptation_lr = config.get("adaptation_lr", 0.03)
+
+        self.batch_size = config.get("batch_size", 5)
+        self.val_batch_size = config.get("val_batch_size", 15)
+        self.slow_batch_size = config.get("slow_batch_size", 64)
+        self.replay_batch_size = config.get("replay_batch_size", 64)
+
+    @classmethod
+    def create_loaders(cls, config):
+        """Create train and val dataloaders."""
+
+        # TODO this method needs to (a) be a class method, and (b) return 2 dataloaders
+
+        dataset_class = config.get("dataset_class", None)
+        if dataset_class is None:
+            raise ValueError("Must specify 'dataset_class' in config.")
+
+        # Create datasets -> same, only initialize two of them
+        dataset_args = config.get("dataset_args", {})
+        dataset_args.update(train=True)
+        main_set = dataset_class(**dataset_args)
+
+        # All loaders share tasks and dataset, but different indices and batch sizes
+        train_slow_loader = cls.create_slow_train_dataloader(main_set, config)
+
+        # For pre/post epoch and batch processing slow loader is equiv to train loader
+
+        # TODO modify return values? right now, it's returning the torch dataloader
```

Comment:
Is this comment really meant to be a `TODO`?

Reply:
Doesn't need to be. I modified it and removed TODO.